src/Sav/Record/Info/VeryLongString.php

```diff
@@ -31,7 +31,7 @@ public function write(Buffer $buffer)
         if ($this->data) {
             $data = [];
             foreach ($this->data as $key => $value) {
-                $data[] = sprintf('%s=%05d%c', $key, $value, 0);
```

Comment:
According to docs the old implementation is correct (from https://www.gnu.org/software/pspp/pspp-dev/html_node/Very-Long-String-Record.html#Very-Long-String-Record)

`char string_lengths[];`
> A list of key–value tuples, where key is the name of a variable, and value is its length. The key field is at most 8 bytes long and must match the name of a variable which appears in the variable record (see Variable Record). The value field is exactly 5 bytes long. It is a zero-padded, ASCII-encoded string that is the length of the variable. The key and value fields are separated by a ‘=’ byte. Tuples are delimited by a two-byte sequence {00, 09}. After the last tuple, there may be a single byte 00, or {00, 09}. The total length is count bytes.